specs/Spec_for_editing_user.adoc

```diff
@@ -0,0 +1,170 @@
+# User Management: Editing User view for Admin
+
+## Problem description
+
+This spec talks about viewing or editing information stored about a single user.
+
+## Use Cases
+
+The flow goes as follows:-
+
+* A user logs in as Admin
+* Clicks on Admin tab in Navigation
+* The navigation drops down
+* Clicks on *Users*
+* Clicks on *Edit* option for a particular user
+
+User will land onto https://redhat.invisionapp.com/share/KNB25OEQT#/screens/226063805[Edit User] view page
+
+## Proposed workflow
+
+### UI
+
+Edit user view consist of a form:-
+
+* ```User ID``` - Not Editable
+* ```Type : Internal``` - Not Editable
+* Status switch - to check the *status* of user
+** Input type: checkbox
+* First Name
+** Input type: text
+* Last Name
+** Input type: text
+* Email
+** Input type: email
+** validation : Input should be a valid email
+* Email Notifications - whether user should receive notifications
+** Input type: checkbox
+* Password
+** Input type: password
+** Validation: Should be minimum of 8 characters
+** Should contain a Show/Hide button for password visibility. The default visibility will be *hidden*.
+* Select a Role
+** Roles: Admin, Normal, Limited
+** The default will be the current assigned role of User
+* ```Save``` button
+** Enabled when User makes change in this form.
+** This button commits changes and returns to User List view page.
```

Comment:
We should show a messages as well that user edit was successful as a balloon I feel. Do we have that?